geometry/geometry_config.py

```diff
@@ -146,13 +146,13 @@
 
      c.xMax = 2 * u.m  # max horizontal width at T4
      #
-     c.TrackStation4 = AttrDict(z=z4)
-     zset=z4-200.*u.cm
-     c.TrackStation3 = AttrDict(z=zset)
-     zset=z4-640.*u.cm-magnetIncrease
-     c.TrackStation2 = AttrDict(z=zset)
-     zset=z4-840.*u.cm-magnetIncrease
-     c.TrackStation1 = AttrDict(z=zset)
+     c.TrackStation4 = AttrDict(z = z4)
+     zset = z4 - 200.*u.cm
```

Comment:
2 m is the separation between two tracking stations on either side of the magnet?

Then let's name this magic number.